cirq/ops/moment.py

```diff
@@ -37,7 +37,7 @@ class Moment:
         qubits: A set of the qubits acted upon by this Moment.
     """
 
-    def __init__(self, operations: Iterable[raw_types.Operation] = ()) -> None:
+    def __init__(self, _operations: Iterable[raw_types.Operation] = ()) -> None:
```

Comment:
Don't change this variable name.

Reply:
@kevinsung  yeah it was for some testing i removed it.

Reply:
@kevinsung I am getting below error on document generation. I understand it's because 

Warning, treated as error:
/home/travis/build/quantumlib/Cirq/docs/generated/cirq.Moment.operations.rst:2:duplicate object description of cirq.Moment.operations, other instance in generated/cirq.Moment, use :noindex: for one of them
FAILED
The command "dev_tools/build-docs.sh" exited with 2.